firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -1029,6 +1032,47 @@ public void testMultipleUpdatesWhileOffline() {
     assertEquals(asList(map("foo", "zzyzx", "bar", "2")), querySnapshotToValues(snapshot2));
   }
 
+  @Test
+  public void resumingQueryShouldRemoveDeletedDocumentsIndicatedByExistenceFilter()
+      throws InterruptedException {
+    // TODO(Mila):Remove this condition once the bug is resolved.
```

Comment:
You can remove this comment: "TODO(Mila):Remove this condition once the bug is resolved" because it is implied by the assumeFalse() line below.